### PR TITLE
[ggj][infra][1/5] integration-test: add goldens files for Redis client library

### PR DESCRIPTION
**Pull request by xiaozhenliu-gg5**
Find more details in design doc: go/java-micro-integration-test-infra

1. add client library files generated from monolith generator (3 intermediary srcjars from [rule](https://github.com/googleapis/gapic-generator-java/blob/master/rules_java_gapic/java_gapic.bzl#L37)) to goldens folder.
2. add a bazel file for the goldens/redis folder.

Next Step:

[2/5] Add integration test bazel rules which will run Redis client library generation and compare Java source code with the added goldens files.

### CODE REVIEW

**Review by miraleung — APPROVED** [review]
Could we add a README to `test/integration/goldens` so that future readers will know that these are actual source Java files, but just goldens?

**Review by miraleung — CHANGES_REQUESTED** [review]
On second thought, let's update these goldens with the upcoming microgenerator rules.

**Comment by xiaozhenliu-gg5** [comment]
On hold until the microgenerator rules are ready and update Redis goldens. @miraleung

File: test/integration/goldens/redis/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "redis_goldens_files",
```
Comment (miraleung) [inline test/integration/goldens/redis/BUILD.bazel:4]: Just `goldens_files` should be sufficient, since this is in the `redis` directory.

**Review by miraleung — APPROVED** [review]
LGTM with BUILD target comment.

File: test/integration/goldens/redis/BUILD.bazel (hunk 1 above)
Comment (xiaozhenliu-gg5) [reply test/integration/goldens/redis/BUILD.bazel:4]: SG, renamed

**Pull request merged**